yarn.lock

```diff
@@ -9035,10 +9035,10 @@ react-clone-referenced-element@^1.0.1:
   resolved "https://registry.yarnpkg.com/react-clone-referenced-element/-/react-clone-referenced-element-1.1.0.tgz#9cdda7f2aeb54fea791f3ab8c6ab96c7a77d0158"
   integrity sha512-FKOsfKbBkPxYE8576EM6uAfHC4rnMpLyH6/TJUL4WcHUEB3EUn8AxPjnnV/IiwSSzsClvHYK+sDELKN/EJ0WYg==
 
-react-devtools-core@^4.12.0:
-  version "4.12.2"
-  resolved "https://registry.yarnpkg.com/react-devtools-core/-/react-devtools-core-4.12.2.tgz#4198f42fec867ed82dd0ee0f059b450e703b2b1e"
-  integrity sha512-cvAiJCSIIan2A22o4j4Twc7PdDrwqiAQVBeZ+osS2T/wv2Ua3a0J8Sgx4pTH5Y7VoWn5WiGCHkAW4S1lYl3kcA==
+react-devtools-core@^4.6.0:
```

Comment:
Just a not: Interesting how we somehow are rolling back the declared dependency, the resolved version increases, so fine... but still what dependency downgraded for 12 to 6?

Reply:
A found it, `react-native@0.0.0-5127c71d6` downgraded from 12 to 6. 
Do you think this warrants an investigation in case there are compatibility issues? Since we override it with 4.12.4?

Reply:
I'll test dev tools manually with the resolution removed.

Reply:
Oh, I see that this is the yarn resolution for upstream semver constraint.

I put a bit of thought into this. I think I'd rather we align to upstream semver contraints as-is, since that it what users of RN will get when they install.